### ISSUE TITLE
test API - print collector

### ISSUE THREAD

**Issue by vesely-david** [PMAlgo/PMWebApp#13]
Andrea please, would it be possible to print to console whole information in edges_collector before return from api function? Don't you have already any implemented method?

**Comment by AndreaCorallo** [comment]
Yes of course. To be called from the API example will require a bit of code moving because outside the api some internal struct required for printing (ex the names of the variables) are not available.
I did some code moving for making these func available to different parts of the code (the func where the one that you leaved in the pmc.c). I also documented it.
Now they are implemented in pmc_learn.c and defined in the pmc_learn.h if you like to take a look.
If you really want to print from the example I have to move these inside the API (where all the data needed for printing are available) and add a print flag to the API func. Would be this ok?
Other option is the return from the API also the feature names etc etc. Maybe this would be needed by you to print the rules?
Or if you like just to see the implementation the function is fprint_rules in https://github.com/PMAlgo/pmc-0.2/blob/master/src/pmc_learn.c

**Comment by vesely-david** [comment]
Actualy what i meant was only for debugging. I would need to completely print every variable from edges_collector and corresponding edges. And I would need to print it in c code to console. Reason is i need to compare the results from c, with result in ruby c code wrapper. I think i found the week place of the ruby wrapper. It looks like i am itterating more times than the pointer is actualy allocated but i have no idea why ( i am iterating len-times).

**Comment by AndreaCorallo** [comment]
Understood!
so in this case is sufficient to run the normal pmc executable with the same parameters :)
./pmc -t0 -C10 -e0 -c5 -m1 -d10 ../datasets/2dchess_1000p_1000v_10dim_0err.float 
Hope that it helps let's me know!
